drivers/sensor/bmm150/bmm150.c

```diff
@@ -475,6 +505,9 @@ static int bmm150_attr_set(const struct device *dev,
 static const struct sensor_driver_api bmm150_api_funcs = {
 #if defined(BMM150_SET_ATTR)
 	.attr_set = bmm150_attr_set,
+#endif
+#ifdef CONFIG_BMM150_TRIGGER
+	.trigger_set = bmm150_trigger_set,
```

Comment:
This change looks unrelated to SPI/I2C

Reply:
> This change looks unrelated to SPI/I2C

I will fix it, and take another PR for trigger mode.